system/make.py

```diff
@@ -90,9 +93,9 @@ def sanitize_output(output):
     :return: sanitized output
     """
     if output is None:
-        return b('')
+        return ''
```

Comment:
I wasn't sure, but I did a test and it did seems fine. But that's also why we have a review.